commit 60ff5923a96acd938ea7ba2bf0fbfddd017175a1
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Thu Mar 5 10:17:51 2015 +0200

    netdev-dpdk: Support DPDK >= 2.0 unified RSS offload types too
    
    RSS offload types were unified and simplified in DPDK 2.0, easily
    handled with an ifdef though.
    
    Signed-off-by: Panu Matilainen <pmatilai@redhat.com>

diff --git a/lib/netdev-dpdk.c b/lib/netdev-dpdk.c
index 1ba8310..b095826 100644
--- a/lib/netdev-dpdk.c
+++ b/lib/netdev-dpdk.c
@@ -97,8 +97,12 @@ static const struct rte_eth_conf port_conf = {
     .rx_adv_conf = {
         .rss_conf = {
             .rss_key = NULL,
+#ifdef ETH_RSS_IPV4_TCP /* dpdk < 2.0 */
             .rss_hf = ETH_RSS_IPV4_TCP | ETH_RSS_IPV4 | ETH_RSS_IPV6
                     | ETH_RSS_IPV4_UDP | ETH_RSS_IPV6_TCP | ETH_RSS_IPV6_UDP,
+#else
+            .rss_hf = ETH_RSS_IP | ETH_RSS_UDP | ETH_RSS_TCP,
+#endif
         },
     },
     .txmode = {
